nalgebra-sparse/src/csc.rs

```diff
@@ -524,6 +520,75 @@ impl<T> CscMatrix<T> {
     }
 }
 
+#[cfg(feature = "serde-serialize")]
+mod serde_serialize {
+    use super::CscMatrix;
```

Comment:
Would be nice to have this stuff in a separate file maybe? The same for other types.